laalaa/apps/advisers/fixtures/initial_categories.json

```diff
@@ -103,22 +103,6 @@
     "model": "advisers.category",
     "pk": 13
   },
-  {
-    "fields": {
-      "civil": true,
-      "code": "DISC"
-    },
-    "model": "advisers.category",
-    "pk": 14
-  },
-  {
-    "fields": {
-      "civil": true,
-      "code": "EDU"
```

Comment:
What do you think about keeping changes in the seed fixtures?
It is still useful to populate these entries on our environments where we don't have uploads -- until we can add a "seed upload".

Reply:
In other words, I think it's enough to revert the migration.